doc/source/NewSiteConfigs.rst

```diff
@@ -642,7 +642,7 @@ Creating a new environment with Nvidia compilers
 .. warning::
    Support for Nvidia compilers is experimental and limited to a small subset of packages of the unified environment. The Nvidia compilers are known for their bugs and flaws, and many packages simply don't build. The strategy for building environments with Nvidia is therefore the opposite of what it is with other supported compilers.
 
-In order to build environments with the Nvidia compilers, a different approach is needed than for our main compilers (GNU, Intel). Since many packages do not build with the Nvidia compilers, the idea is to provide as many packages as possible as external packages or build them with ``gcc``. Because our spack extension ``spack stack setup-meta-modules`` does not support combiniations of modules built with different compilers, packages not being built with the Nvidia compilers need to fulfil the two following criteria:
+In order to build environments with the Nvidia compilers, a different approach is needed than for our main compilers (GNU, Intel). Since many packages do not build with the Nvidia compilers, the idea is to provide as many packages as possible as external packages or build them with ``gcc``. Because our spack extension ``spack stack setup-meta-modules`` does not support combinations of modules built with different compilers, packages not being built with the Nvidia compilers need to fulfil the two following criteria:
```

Comment:
This is actually no longer true. We now do support building certain packages with a different compiler. We use this with intel/oneapi as preferred compilers, for example, where gcc is the fallback. But, for each of the packages that we use the fallback compiler, we need to check carefully if that is ok (e.g. it can't be a package that depends on MPI, for example). If this is helpful, I can talk you through this next week.

Reply:
Ah, interesting! I would like to chat and better understand the implications of this. I will not have any time next week, but perhaps on the week of the 27th?

Reply:
Sounds good, please ping me on slack (I will forget, 100% guaranteed)

Reply:
Dom and I decided I'd update the comment to indicate this instruction set is out-of-date with the newest capabilities to use multiple compilers. But we'll defer the work to _use_ multiple compilers until after the 1.9 release, because that will be too difficult to wrap up in time to include in the release.

PR incoming.